lib/vorpal.js

```diff
@@ -1057,6 +1058,31 @@ vorpal._commandHelp = function (command) {
     ['', '  Invalid Command. Showing Help:', ''].join('\n') :
     '';
 
+  if ('' !== invalidString) {
+    var catchCmd = false;
+    for (var i = 0; i < this.commands.length; i++) {
+      catchCmd = this.commands[i]._catch;
+      if (catchCmd) {
+        break;
+      }
+    }
+
+    if (! catchCmd) {
+      var hints = [];
+      for (var i = 0; i < this.commands.length; i++) {
+        if (4 > leven(command, this.commands[i]._name)) {
+          hints.push(this.commands[i]._name);
+        }
+      }
+
+      if (0 < hints.length) {
+        return  '\n  ' + command +
+                ' is an invalid command. Maybe you mean:\n\n    ' +
+                hints.join('\n    ');
+      }
+    }
+  }
+
```

Comment:
This can be greatly simplified with es6 syntax.

eg (untested!):
```js
if (invalidString !== ''){

    let catchCmd = this.commands.find(cmd => cmd._catch);

    if (!catchCmd){
        
        let hints = this.commands.reduce((foundHints, cmd) => {
            if (leven(command, cmd._name) < 4) {
                foundHints.push(cmd._name);
            }
            return foundHints;
        }, []);
        
        if (hints.length){
            return `\n  ${command} is an invalid command. Maybe you mean:\n\n    ${hints.join('\n    ')}`;
        }
    }
}
```